container-search/src/main/antlr4/com/yahoo/search/yql/yqlplus.g4

```diff
@@ -459,10 +459,14 @@ propertyNameAndValue
 	;
 
 propertyName
-	: IDENTIFIER
```

Comment:
This no longer matches `propertyref`, but the similar names don't _necessarily_ mean they are similar. Is this OK?

Reply:
Yes, forget I asked!